modules/core/src/main/scala/org/scalasteward/core/buildtool/scalacli/ScalaCliAlg.scala

```diff
@@ -28,6 +28,27 @@ import org.scalasteward.core.io.{FileAlg, ProcessAlg, WorkspaceAlg}
 import org.scalasteward.core.util.Nel
 import org.typelevel.log4cats.Logger
 
+object ScalaCliAlg {
+  val directives =
+    // sourced from https://github.com/VirtusLab/scala-cli/blob/9e22d4a91ba8699ac2727d2ac3042d64abe951e1/modules/directives/src/main/scala/scala/build/preprocessing/directives/Dependency.scala#L33-L48
+    List(
+      "lib",
+      "libs",
+      "dep",
+      "deps",
+      "dependencies",
+      "test.dependency",
+      "test.dep",
+      "test.deps",
+      "test.dependencies",
+      "compileOnly.lib",
+      "compileOnly.libs",
+      "compileOnly.dep",
+      "compileOnly.deps",
+      "compileOnly.dependencies"
+    ).map(alias => s"//> using $alias ")
```

Comment:
@bishabosha do we need `toolkit` in this list as well? So we can get updates for directives like
```
//> using toolkit 0.2.1
```

Reply:
From what I could see this would need a bigger change as the algorithm searches for explicit strings so unless you want to search for all possible toolkits this isn't very flexible

Reply:
@bishabosha thanks for looking at it but I'm not sure I understand what you mean. Are you saying that adding `toolkit` here _wouldn't_ match
```
//> using toolkit typelevel:0.1.19
```
why not?

Reply:
Scala Steward can already update toolkits: https://github.com/scala-steward-org/test-repo-1/pull/82/files
So I think it makes sense to add `//> using toolkit` here so that projects, that have no other dependencies than a toolkit, are recognized as Scala CLI projects.